server/lib/systemd/pbench-server.service.rhel7

```diff
@@ -10,8 +10,10 @@ After=network.target httpd.service
 
 [Service]
 Type = simple
-ExecStart = /usr/bin/scl enable rh-python36 -- /opt/pbench-server/bin/pbench-server
-ExecStop = /usr/bin/pkill pbench-server
+PIDFile = /run/pbench-server.gunicorn.pid
```

Comment:
I probably should have made this a template as well and avoid having separate files for RHEL7 and everything else.

Reply:
I can probably do it using the ansible python api [ansible python api](https://docs.ansible.com/ansible/latest/dev_guide/developing_api.html#) (making a decision on whether to use scl or not by checking the ansible distribution) but I havent found a way to get ansible variables like `ansible_distribution_major_version` or `ansible_distribution` via python api

Reply:
I'd rather it be done at installation time. No need to complicate this part of the code.